### PR TITLE
Refactor Top AVS, Operators and stats

### PR DESCRIPTION
**Pull request by Zishan-7**

### CODE REVIEW

File: src/home/Home.jsx [hunk 1]
```diff
@@ -1,9 +1,45 @@
...
+        dispatch({
+          isFetchingEigenlayerTVL: false,
+          error: handleServiceError(e)
+        });
+      }
+    }
+
+    fetchEigenlayerTVL();
+  }, [dispatch, eigenlayerService]);
+
```
Comment (Zishan-7) [inline src/home/Home.jsx:35]: Added this here to save multiple API calls. We can pass this data to Eigenlayer TVL chart component through props.

File: src/home/OverviewStats.jsx [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+            eigenlayerTVL.length > 0 &&
+            state.rate > 1 && (
+              <span className="text-center">
+                <span className="text-sm text-success">
+                  {formatUSD(calculateEigenlayerTVL() * state.rate, compact)}
+                </span>
+              </span>
+            )}
+        </div>
+        <div className="h-fusll flex min-h-10 basis-1/3 flex-col items-center gap-2 border-x border-outline px-2">
```
Comment (rubo) [inline src/home/OverviewStats.jsx:133]: `h-fusll` not needed even without typo :)

File: src/home/OverviewStats.jsx [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+            <ErrorMessage message="Failed loading Total AVS" />
+          )}
+
+          {!state.isFetchingAVS && state.avs.length > 0 && (
+            <span className="text-lg text-white md:text-2xl">
+              {formatNumber(state.totalAVS)}
+            </span>
+          )}
+        </div>
+        <div className="flex basis-1/3 flex-col items-center gap-1">
```
Comment (rubo) [inline src/home/OverviewStats.jsx:153]: The gap here is 1 while for "Total AVS" is 2.

**Review by rubo — CHANGES_REQUESTED** [review]

File: src/home/Home.jsx (hunk 1 above)
Comment (rubo) [reply src/home/Home.jsx:35]: Good but please change `Eigenlayer` to `EigenLayer`.

File: src/home/OverviewStats.jsx (hunk 2 above)
Comment (Zishan-7) [reply src/home/OverviewStats.jsx]: Oops! left it there by mistake.

Comment (rubo) [reply src/home/OverviewStats.jsx]: Well, turned out it wasn't a mistake. We need `h-full justify-center` on all of them not to have too short border:

image

**Review by rubo — CHANGES_REQUESTED** [review]
Let's fix paddings for skeletons and we're good for now.

Screenshot 2024-07-30 212946

File: src/home/OverviewStats.jsx [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+            state.rate > 1 && (
+              <span className="text-center">
+                <span className="text-sm text-success">
+                  {formatUSD(calculateEigenLayerTVL() * state.rate, compact)}
+                </span>
+              </span>
+            )}
+        </div>
+        <div className="flex min-h-10 basis-1/3 flex-col items-center gap-1 border-x border-outline px-2">
+          <span className="text-center text-xs text-foreground-1 md:text-sm">
```
Comment (rubo) [inline src/home/OverviewStats.jsx:130]: `text-center` looks redundant

File: src/home/OverviewStats.jsx [hunk 5]
```diff
@@ -0,0 +1,398 @@
...
+              classNames={{ base: 'h-6 w-28 rounded-md border-none' }}
+            />
+          )}
+
+          {!isFetchingEigenlayerTVL && eigenLayerTVLError && (
+            <ErrorMessage message="Failed loading EigenLayer TVL" />
+          )}
+
+          {!isFetchingEigenlayerTVL && eigenLayerTVL.length > 0 && (
+            <span className="text-center">
```
Comment (rubo) [inline src/home/OverviewStats.jsx:112]: This `span` looks redundant.

**Review by rubo — CHANGES_REQUESTED** [review]

File: src/home/OverviewStats.jsx [hunk 6]
```diff
@@ -0,0 +1,398 @@
...
+  );
+}
+
+function TopAVS({ isLoading, avs, rate, error }) {
+  const navigate = useNavigate();
+  const columns = [
+    {
+      key: 'avs',
+      label: 'AVS',
+      className: 'w-64 md:w-2/5 ps-12'
```
Comment (rubo) [inline src/home/OverviewStats.jsx:192]:
```suggestion
      className: 'w-64 md:w-2/5 ps-4'
```

File: src/home/OverviewStats.jsx [hunk 7]
```diff
@@ -0,0 +1,398 @@
...
+  );
+}
+
+function TopOperators({ isLoading, operators, rate, error }) {
+  const navigate = useNavigate();
+  const columns = [
+    {
+      key: 'operator',
+      label: 'Operator',
+      className: 'w-64 md:w-2/5 ps-12'
```
Comment (rubo) [inline src/home/OverviewStats.jsx:299]:
```suggestion
      className: 'w-64 md:w-2/5 ps-4'
```

**Review by rubo — CHANGES_REQUESTED** [review]

File: src/home/OverviewStats.jsx (hunk 2 above)
Comment (Zishan-7) [reply src/home/OverviewStats.jsx]: According to the designs it has short borders 🤔

Screenshot 2024-07-31 at 1 38 34 PM

**Review by rubo — APPROVED** [review]

**Pull request merged**